packages/manager/modules/at-internet-configuration/src/provider.js

```diff
@@ -0,0 +1,32 @@
+export default class atInternetConfigurationProvider {
+  constructor() {
+    this.stateRules = [];
+    this.extraConfig = {};
+    this.config = {};
+  }
+
+  setConfig(config) {
+    this.config = config;
+  }
+
+  setReplacementRules(rules) {
+    this.stateRules = rules;
+  }
+
+  setPrefix(prefix) {
+    this.prefix = prefix;
+  }
+
+  $get() {
```

Comment:
Missing annotation I think.

```suggestion
    /* @ngInject */
    $get() {
```

Reply:
Nothing is passed as parameter so nothing needs to be injected here